continuous_integration/environment-3.10-gpu-dev.yaml

```diff
@@ -0,0 +1,50 @@
+name: dask-sql
+channels:
+  - rapidsai
+  - nvidia
+  - conda-forge
+  - nodefaults
+dependencies:
+  - dask-ml=2022.1.22
+  - dask=2022.3.0
+  - fastapi=0.69.0
+  - intake=0.6.0
+  - jsonschema
+  - lightgbm
+  - maturin=0.12.8
+  - mlflow
+  - mock
+  - nest-asyncio
+  - pandas=1.1.2
+  - pre-commit
+  - prompt_toolkit
+  - psycopg2
+  - pyarrow=3.0.0
+  - pygments
+  - pyhive
+  - pytest-cov
+  - pytest-xdist
+  - pytest
+  - python=3.10
+  - rust=1.60.0
+  - scikit-learn=1.0.0
+  - setuptools-rust=1.1.2
+  - sphinx
+  - tpot
+  - tzlocal=2.1
+  - uvicorn=0.11.3
+  # fugue dependencies; remove when we conda install fugue
+  - adagio
+  - antlr4-python3-runtime<4.10
+  - ciso8601
+  - fs
+  - pip
+  - qpd
+  - triad
+  # GPU dependencies
+  - cudatoolkit=11.5
+  - cudf=22.06
+  - dask-cudf
+  - dask-cuda
```

Comment:
We'll probably want to add the other GPU packages installed in the image to this list:

https://github.com/rapidsai/dask-build-environment/blob/8da1d032d855d487f760749f8bffeb1e532caea9/dask_sql.Dockerfile#L25-L35